www/src/components/thumbprint-components/component-header/index.jsx

```diff
@@ -2,6 +2,18 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import PlatformNav from './platform-nav';
 import ComponentPackageTable from './component-package-table';
+import Alert from '../../alert';
+
+const UIKitDeprecatedCopy = () => {
+    return (
+        <Alert type="warning" title="UIKit based Thumbprint is no longer being developed">
```

Comment:
This is headed in the right direction, but this still feels a bit heavy. I'm wondering if we can come up with a pithier approach to this messaging and expand on what we have here a bit more in the November release notes.

Reply:
@egoens Sure.  Do you have any suggestions on what the copy should be?  

And @danoc do you have any suggestion on how to implement this without `window.location.pathname` which doesn't appear to be compatible with the documentation generator (if I'm reading that error message correctly).

Reply:
@dwroth I'm currently floating this message with our content team, but I'm also making a proposal to them as well. Here is my suggestion for shortening this verbiage:

> Thumbprint is transitioning our codebase from UIKit to SwiftUI, and this documentation is no longer being supported. Please refer to the SwiftUI documentation when available.

Reply:
Oh, I like that a lot! 👍

Reply:
Checked with CD for the compy here and got the 👍🏼